cpr/session.cpp

```diff
@@ -486,6 +486,17 @@ void Session::Impl::SetSslOptions(const SslOptions& options) {
         if (!options.key_pass.empty()) {
             curl_easy_setopt(curl_->handle, CURLOPT_KEYPASSWD, options.key_pass.c_str());
         }
+    } else if (!options.key_blob.empty()) {
+      struct curl_blob blob;
```

Comment:
```suggestion
      curl_blob blob{};
```
Should fix the clang tidy issue and since we are using C++, we don't need the `struct` keyword here.

Reply:
Is there a way to run checks by hand using cmake?

Reply:
In case you have clang-tidy installed on your system you can do something like this:
```bash
mkdir build
cd build
cmake -DCPR_ENABLE_LINTING=ON ..
cmake --build .
```

Reply:
Here you can find a list of all options available: https://github.com/libcpr/cpr/blob/da381bb2afe4e36b18b1fe206bc06951d8fdc06e/CMakeLists.txt#L36-L49